llama_index/query_engine/transform_query_engine.py

```diff
@@ -25,10 +25,12 @@ def __init__(
         query_engine: BaseQueryEngine,
         query_transform: BaseQueryTransform,
         transform_extra_info: Optional[dict] = None,
+        **kwargs: Any,
     ) -> None:
         self._query_engine = query_engine
         self._query_transform = query_transform
         self._transform_extra_info = transform_extra_info
+        super().__init__(**kwargs)
```

Comment:
I think you'll notice I was split on passing in the callback manager directly (and therfore adding imports/types to the constructor of every query engine), or just abusing kwargs lol

Reply:
The issue with passing kwargs is:
1. opaque: not clear what is being accepted by the super class
2. brittle: if we want the subclass to silently ignore kwargs, right now would break since we pass all kwargs into super, which only accepts callback manager.
3.

Reply:
Done!